source/adapter.screenshare.js

```diff
@@ -75,9 +74,9 @@
     };
 
   } else if (window.navigator.webkitGetUserMedia) {
-    baseGetUserMedia = window.navigator.getUserMedia;
 
     navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      baseGetUserMedia = window.navigator.getUserMedia;
```

Comment:
@johache that's strange, since I'm using Chrome 53 (Version `53.0.2785.70` beta  specifically) with the current live [getaroom.io](https://getaroom.io) and it works for me. The live version uses AdapterJS screenshare version `0.13.3`.

I have also tested with various functions `navigator.mediaDevices.getUserMedia`, `navigator.getUserMedia` and `window.getUserMedia` and it works with retrieving stream or screensharing stream in getaroom.io.

Here's an excerpt of the getaroom.io code I modified for testing:
```
Skylink.init({
  roomServer: Configs.Skylink.roomServer,
  apiKey: useMCU ? Configs.Skylink.apiMCUKey : Configs.Skylink.apiNoMCUKey,
  defaultRoom: room
}, function() {
  navigator.getUserMedia({
     audio: true,
     video: true
}, function (stream) {
   window.console.log('Got stream', stream);
   Skylink.sendStream(stream);
      Skylink.joinRoom();
   }, function (error) {
      window.console.error('The error', error);
   });
});
```
This error seems familiar and I wonder if it's related to the error I received, in which I close Chrome and re-opened it again, and then it somehow retrieved the stream. 

Additionally, in regards to `sudo npm install` to update the `third_party` modules, I am using a different PC which probably had some permission restrictions and looks like not being able to do `sudo npm install` to update the dependencies makes it painful, since I get the `npm ERR! Error: EACCES: permission denied, mkdir '/Users/leticiachoo/Workspace/AdapterJS/node_modules/.staging'` errors.